### PR TITLE
enh(install): update platform registration documentation and translation

### PR DESCRIPTION
**Pull request by jeremyjaouen**
## Description

Update documentation to use the new registration script.

## Target serie

- [ ] 20.04.x
- [x] 20.10.x
- [x] 21.04.x (master)

### CODE REVIEW

**Comment by sc979** [comment]
@jeremyjaouen , don't forget to cherry-pick on the 20.10.x branch after your merge ;)

File: en/migrate/poller-display-to-remote-server.md [hunk 1]
```diff
@@ -47,118 +47,125 @@ Otherwise, refer to the [migration procedure](../migrate/migrate-from-3-4.html).
...
 
-    ``` shell
-    2020-10-20T10:39:30+02:00 [ERROR]: Can’t connect to the API using: https://192.169.0.1:443/centreon/api/latest/login
-    ```
+``` shell
+2020-10-20T10:39:30+02:00 [ERROR]: Can’t connect to the API using: https://192.168.0.1:443/centreon/api/latest/login
+```
 
-    > The access url is not complete or invalide. Use the **--root** option to define the API URL Path. For example: **--root monitoring**.
+> The access url is not complete or invalide. Use the **--root** option to define the API URL Path. For example: **--root monitoring**.
```
Comment (sc979) [inline en/migrate/poller-display-to-remote-server.md:162]:
```suggestion
> The access url is not complete or invalid. Use the **--root** option to define the API URL Path. For example: **--root monitoring**.
```

File: en/installation/installation-of-a-remote-server/using-centreon-iso.md [hunk 2]
```diff
@@ -240,13 +248,13 @@ You will receive the validation of the Centreon central server:
...
 ```
 
 > Unable to access to the API. Please check **<IP_TARGET_NODE>**, scheme and port.
 
 ``` shell
-2020-10-20T10:39:30+02:00 [ERROR]: Can’t connect to the API using: https://192.169.0.1:443/centreon/api/latest/login
+2020-10-20T10:39:30+02:00 [ERROR]: Can’t connect to the API using: https://192.168.0.1:443/centreon/api/latest/login
 ```
 
 > The access url is not complete or invalide. Use the **--root** option to define the API URL Path. For example: **--root monitoring**.
```
Comment (sc979) [inline en/installation/installation-of-a-remote-server/using-centreon-iso.md:260]:
```suggestion
> The access url is not complete or is invalid. Use the **--root** option to define the API URL Path. For example: **--root monitoring**.
```

**Comment by sc979** [comment]
Branch rebased and conflits resolved

**Review by sc979 — APPROVED** [review]

File: en/installation/installation-of-a-poller/using-centreon-iso.md [hunk 3]
```diff
@@ -162,61 +156,42 @@ Example:
...
-
-    proxy host: myproxy.example.com
-
-    proxy port: 3128
-
-    proxy username (press enter if no username/password are required): myuser
-
-    please enter the proxy password:
-    ```
+``` shell
```
Comment (lpinsivy) [inline en/installation/installation-of-a-poller/using-centreon-iso.md:159]: please re-add the tab/indentation becofe code for better readability

**Review by lpinsivy — APPROVED** [review]

**Pull request merged**